app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc.rb

```diff
@@ -76,6 +80,14 @@ def tags_by_crn(crn)
     connection.cloudtools.tagging.collection(:list_tags, :attached_to => crn, :providers => ["ghost"]).to_a
   end
 
+  def resource_instances
+    @resource_instances ||= connection.cloudtools.resource.controller.collection(:list_resource_instances)
+  end
+
+  def database_instances
+    @database_instances ||= resource_instances.select { |res| res[:resource_plan_id].match?(/databases-for-*/)}
```

Comment:
Super minor, but this match can also just be a simpler `.include?`

<suggestion>
    @database_instances ||= resource_instances.select { |res| res[:resource_plan_id].include?("databases-for-") }
</suggestion>

Reply:
Nice, I'll make note of this and include it in my next database PR